worlds/sm/Options.py

```diff
@@ -52,6 +52,14 @@ class DeathLink(Choice):
     alias_true = 1
     default = 0
 
+class RemoteItems(Toggle):
+    """Indicates you get items sent from your own world. This allows coop play of a seed."""
```

Comment:
*technically* a seed is the entire multiworld. Since we call it. I think it'd be better to say world here.